subsys/bluetooth/controller/Kconfig

```diff
@@ -196,11 +196,12 @@ comment "BLE Controller features"
 if BT_CONN
 
 config BT_CTLR_LE_ENC
-	bool
+	bool "LE Encryption"
```

Comment:
I will keep this feature hidden. This was added for the only reason to use/test 251 byte PDUs and proprietary 2M PHY in nRF51 without encryption (due to h/w limitation to perform 251 byte CCM or 2M PHY encryption).

Do you really want to have BLE without encryption as an end-user option?

Reply:
I think the core issue here are the Kconfig warnings @sjanc mentions in the commit message. I suspect those are coming from various mesh samples that have CONFIG_BT_CTLR_LE_ENC=n in their prj.conf files to minimize footprint (Mesh has no use for encryption, even over GATT, since all data going over it is encrypted using Mesh's own mechanisms, same as over advertising)